Makefile

```diff
@@ -3,6 +3,15 @@ CONTAINER_ENGINE ?= podman
 CHECKUP_IMAGE_NAME := quay.io/kiagnose/kubevirt-rt-checkup
 CHECKUP_IMAGE_TAG ?= devel
 
+GO_IMAGE_NAME := docker.io/library/golang
+GO_IMAGE_TAG := 1.19.4-alpine3.17
```

Comment:
Shouldn't we use the builder image?

Reply:
This is the same image as the `builder` image.
I'm considering to not use a multi-stage build, in order to have this image only in one place and not two.

Reply:
I've removed the multi-stage build in the Dockerfile.